src/components/Atoms/Controls/Input/components/DefaultInput.tsx

```
@@ -0,0 +1,88 @@
+  disabled?: boolean;
+  isError?: boolean;
+  type: InputType;
+  handleChange?: (
+    event: React.ChangeEvent<HTMLInputElement> | CustomChangeEvent,
+  ) => void;
+  inputRef: (e: HTMLInputElement | null) => void;
+  className?: string;
+  withIcon?: boolean;
+  parentRef: React.ForwardedRef<HTMLInputElement>;
+  children?: React.ReactNode;
+  restProps?: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)